sdk/storage/storage-queue/package.json

```diff
@@ -116,11 +116,10 @@
     "@azure/core-paging": "^1.1.1",
     "@azure/core-rest-pipeline": "^1.10.1",
     "@azure/core-tracing": "^1.0.0",
-    "@azure/core-util": "^1.1.1",
+    "@azure/core-util": "^1.6.1",
```

Comment:
I don't see other updates in this library and I wonder if it actually needs uuid/core-util?